### DISCUSSION TITLE
Questionable Choices

### DISCUSSION THREAD

**Discussion by ItWorkedLastTime** [Snark-Industries/CenterStage#2] [General]
Hello, and welcome to Questionable Choices, here we make, well, questionable choices
this is a place for good ideas, bad ideas, worse ideas, world bad ideas, brain farts, jokes, I really do not care.

rules(so maybe I do care... only a little)

- be nice, no threats, slurs or swears, if you have to swear do something like this: #%$@!
- no criminal plots, this includes but is not limited to, theft, robbery, larceny, acts of blatant terrorism, or plots to steal the moon
- what is said here stays here if the person requests, this is a safe space, where we respect each others wishes and privacy

have fun you sanity-lacking children

**Comment by OrioMann** [comment]
Hell no mua hahaha

**Comment by OrioMann** [comment]
I wonder if theres a cap to how many comments i can make

**Comment by OrioMann** [comment]
Only one way to find out

**Comment by OrioMann** [comment]
What are you typing

**Reply by ItWorkedLastTime** [reply]
W H A T
H O W  Y O U  R E S P O N D  S O  F A S T

**Comment by OrioMann** [comment]
The war starts

**Comment by OrioMann** [comment]
会乐极生悲上班呢很舍不得那么完美

**Comment by OrioMann** [comment]
你的生活真的是太呃好喜欢的1⃣️。你也额没办法看的电影真的不一样了、

**Comment by ItWorkedLastTime** [comment]
WHAT THE #$@!

**Comment by OrioMann** [comment]
很无语弄swearing

**Comment by ItWorkedLastTime** [comment]
HELP

**Comment by OrioMann** [comment]
No swearing

**Comment by OrioMann** [comment]
No no snitching

**Comment by OrioMann** [comment]
😡

**Comment by OrioMann** [comment]
Are you making a new discussion

**Comment by ItWorkedLastTime** [comment]
hey everyone are you all wrapping up???

**Comment by ItWorkedLastTime** [comment]
![hqdefault](https://github.com/Snark-Industries/CenterStage/assets/145297378/e4d18a50-8628-4cd5-95a1-4aa58fb9a7ae)
![thats_it](https://github.com/Snark-Industries/CenterStage/assets/145297378/932ad520-0607-48d1-9004-4d9a636e76af)

**Comment by ItWorkedLastTime** [comment]
/kill OrioMann



shoot this isn't Minecraft
also is your name have Mann like Mann Co??
*tf2 theme starts playing*
